kornia/contrib/models/rt_detr/post_processor.py

```diff
@@ -61,10 +61,8 @@ def forward(self, logits: Tensor, boxes: Tensor, original_sizes: Tensor) -> Tens
         cxcy, wh = boxes[..., :2], boxes[..., 2:]
         boxes_xy = concatenate([cxcy - wh * 0.5, wh], -1)
 
-        sizes_wh = torch.empty(1, 1, 2, device=boxes.device, dtype=boxes.dtype)
-        sizes_wh[..., 0] = original_sizes[0][1]
-        sizes_wh[..., 1] = original_sizes[0][0]
-        sizes_wh = sizes_wh.repeat(1, 1, 2)
+        # Get dynamic size from the input tensor itself
```

Comment:
let's add a todo or open ticket to update with a conversion free function

Reply:
it is done here. Just a note

Reply:
I mean something to be used from Kornia.geometry.boxes